buddyboss/buddyboss-platform
# Changed user suggestion list value to use user_nicename

PROD-1819 #comment fixed the suggession list value to take the user_nicename as in further execution the get_user_by('slug', ...); is used

### All Submissions:




### How to test the changes in this Pull Request:

1. Go to any group and take the user not in the group and make sure the username and nickname should be different.
2. Try to add him to the group from the admin group edit page.
3. Now it should be added

### Proof Screenshots or Video

https://somup.com/c3VI1KZbnl

### Other information:




### Changelog entry

>  Changed user suggestion list value to use user_nicename
